src/stylesheets/utility/_links.scss

```diff
@@ -0,0 +1,22 @@
+a, sup {
+  color: color(coral-pink);
+}
+
+sup {
+  border-bottom: solid rem(1) color(coral-pink);
+  font-family: $base;
+  font-size: rem(11);
+  vertical-align: top;
```

Comment:
I was gonna comment on the superscript numbers not being high enough, but that's how they are in Figma. 👍